### DISCUSSION TITLE
Slow down audio tempo to gain more accurate timestamps

### DISCUSSION THREAD

**Discussion by misutoneko** [linto-ai/whisper-timestamped#128] [Ideas]
Hi, just sharing an observation from my testing with medium model.
Subtitle duration can sometimes get quite excessive (almost 10 secs per subtitle).
Ideally we'd like them to stay at a few secs per subtitle (maybe 4-5s. max?).
There might even be audible gaps that are nowhere to be seen in the .words.* files.

I noticed that ffmpeg's atempo parameter can help with this.
It needs to be set to about 0.75 (or perhaps even lower).
The resulting timestamps need to be adjusted accordingly.

Here's the command:
`ffmpeg -i clip_original.wav -filter:a "atempo=0.75" -vn clip_slow.wav`

The processing will be somewhat slower, but not that bad.
In fact, I remember that whisper.cpp had an opposite idea at some point, to speed up processing by increasing tempo.

**Comment by dgoryeo** [comment]
Hi @misutoneko , I just came across this post. Do I understand correctly that: in your experience slowing down the audio will produce better subtitles? Also, is that language dependent or applicable to any source language? Thanks

**Reply by misutoneko** [reply]
Yes, but I think the improvement only concerns the **timestamps** as the audible gaps in between the spoken words will become more evident.
Since I wasn't able to get a similar improvement with any other method, I considered it to be interesting enough to post here.
So while the method itself is model-agnostic, I suspect it doesn't give you any better WER results for the smaller models if that's what you were thinking :D
